mapfile.c

```diff
@@ -6528,7 +6561,7 @@ int msUpdateMapFromURL(mapObj *map, char *variable, char *string)
           msyystring = string;
           msyylex();
 
-          if(getDouble(&(rotation_angle)) == -1) break;
+          if(getDouble(&(rotation_angle), MS_NUM_CHECK_RANGE, -360, 360) == -1) break;
```

Comment:
As above, I was unaware you could use negative angles (to presumably set anti-clockwise rotation?)

Reply:
Would be good to get confirmation from @tbonfort but this should be ok and should be easy to test.